ghc-app/static/index.html

```diff
@@ -137,7 +137,7 @@
       </div>
     </nav>
 
-    <script type="text/javascript" src="http://l2.io/ip.js?var=clientAddr">
+    <script type="text/javascript" src="https://l2.io/ip.js?var=clientAddr">
```

Comment:
Use `://l2.io/ip.js?var=clietnAddr`, and generally use :// as a protocol for all fully qualified asset references to avoid having to swap schemes.